connexta/ddf-registry
# DDF-2044 Added RegistryPublicationHandler to send registry updates to…

#### What does this PR do?
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@mcalcote @gordocanchola @vinamartin @brianfelix 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
#### How should this be tested?

Unit tests pass. Manually start up two instances and publish one to the other. Update registry values on the one that published and verify that the other system gets the updates.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:

… remote registry the local registry has published to
